src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -316,6 +316,9 @@
     <ReferenceFromRuntime Include="System.Private.Interop" />
     <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
     <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj" />
+    <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj">
+      <Aliases>System_Security_Principal</Aliases>
```

Comment:
Could you please merge the non-uapaot and uapaot blocks? They should be the same.

System.Private.Interop may be exception - if it is the case, include it with a local condition.

mscorlib and Windows should not be needed.